docs/quickstart/aws.md

```diff
@@ -0,0 +1,164 @@
+```
+
+Congrats you’ve SSH’d into the server.
+
+In this instance, Coder can be accessed at the url https://fccad1b6c901511b30cf2cf4fbd0973e.pit-1.try.coder.app.
+
+Copy the URL and run the following command to create the workspace owner:
+
+```sh
+coder login <url***.try.coder.app>
+```
+
+Fill out the prompts - be sure to save use email and password - these are your admin username and password.
+
+You can now access Coder on your local machine with the IP address from your security group by navigating to the `***.try.coder.app` URL and logging in with the username and password.
+
+## Install Coder
+
+For this instance, we will run Coder as a service command, however you can run Coder a multitude of different ways - you can learn more about those here.
+
+In the EC2 instance, run the following command to install Coder
+
+```sh
+curl -fsSL <https://coder.com/install.sh> | sh
+```
+
+## Run Coder
+
+First, edit the `coder.env` file to enable `CODER_TUNNEL` by setting the value to true with the following command:
+
+```sh
+sudo vim /etc/coder.d/coder.env
+```
+<img src="../images/quickstart/aws/aws7.png">
+
+Exit vim and run the following command to start Coder as a system level service:
+
+```sh
+sudo systemctl enable --now coder
+```
+
+The following command will get you information about the Coder launch process
+
+```sh
+journalctl -u coder.service -b
+```
+
+This will return a series of Coder logs, however, embedded in the launch is the URL for accessing Coder.
+
+<img src="../images/quickstart/aws/aws8.png">
+
+In this instance, Coder can be accessed at the url `https://fccad1b6c901511b30cf2cf4fbd0973e.pit-1.try.coder.app`.
+
+Copy the URL and run the following command to create the first user, either on your local machine or in the AWS EC2 instance terminal. 
+
+```sh
+coder login <url***.try.coder.app>
+```
+
+Fill out the prompts - be sure to save use email and password - these are your admin username and password.
+
+You can now access Coder on your local machine with the relevant  `***.try.coder.app` URL and logging in with the username and password.
+
+## Creating and Uploading Your First Template
+
+First, run `coder template init` to create your first template. You’ll be given a list of possible templates to use. This tutorial will show you how to set up your Coder instance to create a linux based machine on AWS. 
+
+<img src="../images/quickstart/aws/aws9.png">
+
+Press `enter` to select `Develop in Linux` on AWS template. This will return the following: 
+
+<img src="../images/quickstart/aws/aws10.png">
+
+To get started using the AWS template, install the AWS CLI and put the credentials in this account. Follow these instructions to install the AWS CLI and add your credentials.
+
+Since we're running Coder as a system level process, a system user `coder` is created in order to run Coder services. The `coder` user will require access to the AWS credentials to initialize the template and provision workspaces.
```

Comment:
```suggestion
The Coder service runs under the `coder` user, so we must authenticate `coder` with AWS before it can initialize the template and provision workspaces.
```

Reply:
we dont actually end up authenticating the coder user but instead just copying and giving it access to the aws credentials.  solution is a blend of the two sentences above

Reply:
I'm confused by what you mean. The copying of the credentials into the `coder` user is authenticating it in my eyes?

Reply:
when i think authentication i think using the established pattern to authenticate a specific user - ie using aws config command when on the coder user to give it credentials - in this case we are copying over the credentials from authenticating the ec2-user over to the coder user

Reply:
Okay, I can see it that way too. My goal was to compress the words as much as possible.

Reply:
How about a compromise? 

> Since the Coder service runs under the `coder` user, we need to move the AWS credentials there.

`aws configure` doesn't really do any authentication either, it's just a utility to generate a text file. The real authentication happens when `coder templates <create/push>` or `coder workspace <create/update>` occurs.

Reply:
regarding word compression my dev rel prerogative is to be as verbose as possible to any possible avoid confusion.

Reply:
There's a middle ground.. After we replace "system level process" with "system service", this paragraph becomes:

```
Since we're running Coder as a system service, a system user `coder` is created in order to run Coder services. The `coder` user will require access to the AWS credentials to initialize the template and provision workspaces.
```

"Since we're running Coder as a system service, a system user `coder` is created in order to run Coder services." I think would need rework.